src/components/Text/textRules.ts

```diff
@@ -24,18 +24,12 @@ import {
 
 import { Sizes } from '../../lib/enums'
 import { truncateStyle } from '../../styles/customCSS'
-import { ITextVariables } from './textVariables'
-
-export interface TextRulesParams {
-  props: any
-  variables: ITextVariables
-}
 
 export default {
   root: ({
     props: { atMention, disabled, error, size, important, success, timestamp, truncated },
     variables: v,
-  }: TextRulesParams) => ({
+  }: any) => ({
```

Comment:
This is also changed in #16, there will be a break out PR that introduces proper baseline typings for styles.